test/controllers/returns/ImportedPlasticPackagingControllerSpec.scala

```diff
@@ -181,19 +189,31 @@ class ImportedPlasticPackagingControllerSpec extends SpecBase with MockitoSugar
 
     "submit must redirect to the mini-cya page if the answer has not changed" in {
 
-      when(mockUserAnswers.change(any(), any())(any())).thenReturn(None) // Respond saying the user-answer hasn't changed
-      when(mockUserAnswers.get(eqq(ImportedPlasticPackagingPage))(any())).thenReturn(Some(true)) // TODO shouldn't need this?
+      // Check change_v3, saveUserAnswerFunc, manufacturedPlasticPackagingRoute, returned redirect val
```

Comment:
Again can we work these comments into the test name and drop the comments?

Reply:
I've done those checks now and removed the comment :+1: